docs/references/cli.md

```diff
@@ -128,6 +128,57 @@ optional arguments:
                      warning, error. Default: info
 ```
 
+### `upgrade`
+
+Upgrades an Opstrace cluster.
+
+NOTE: GCP platform it not yet supported.
+
+Example (to upgrade a cluster named `testcluster` in AWS):
+
+```text
+./opstrace upgrade aws testcluster -c config.yaml
+```
+
+Help text:
+
+```text
+$ ./opstrace upgrade --help
+usage: opstrace upgrade [-h] [--log-level LEVEL] [-c CONFIG_FILE_PATH] [--yes] [--region REGION]
+                        PROVIDER CLUSTER_NAME
+
+positional arguments:
+  PROVIDER              The cloud provider to act on (aws, gcp).
+  CLUSTER_NAME          The Opstrace cluster name ([a-z0-9-_], no more than 23 characters).
+
+optional arguments:
+optional arguments:
+  -h, --help         show this help message and exit
+  --log-level LEVEL  Set log level for output on stderr. One of: debug, info,
+                     warning, error. Default: info
+  -c CONFIG_FILE_PATH, --cluster-config CONFIG_FILE_PATH
+                        File path to cluster config document (YAML). Read
+                        from stdin otherwise.
+  --yes              Automatic yes to prompts; assume 'yes' as answer to all
+                     prompts andrun non-interactively
+  --region REGION    Set the AWS region to destroy in. Only needed when the
```

Comment:
"to destroy in" is not what we want :).

Maybe more of this needs to be adjusted?

Reply:
yep, copy pasted to keep the formattiing and forgot to change.